core/aws-core/src/main/java/software/amazon/awssdk/awscore/AwsServiceClientConfiguration.java

```diff
@@ -55,12 +61,20 @@ public IdentityProvider<? extends AwsCredentialsIdentity> credentialsProvider()
         return credentialsProvider;
     }
 
+    /**
+     * @return The configured map of auth schemes.
+     */
+    public Map<String, AuthScheme<?>> authSchemes() {
```

Comment:
I may be misremembering, but weren't we saying this should be on the SdkServiceClientConfiguration since it is generic and not aws-specific?

Reply:
Good catch, I opened #4580 to fix this and other issues. Thanks!